### PR TITLE
docs: Actualized TRANSLATION guide

### PR DESCRIPTION
**Pull request by alexandrtovmach**
Actualized TRANSLATION guide

@nodejs/i18n 
@nodejs/i18n-api 
@nodejs/website

### CODE REVIEW

File: TRANSLATION.md [hunk 1]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
-    nodejs-ro: Add files - 15 files
-
-    nodejs-fr: Update files - es6.md
-    nodejs-ja: Update files - all files
-```
-
-* Do not make language specific changes to layout or styling in a translation PR. If they are needed, make a separate styling/layout pr and talk with one of the website WG about the change. We want to make sure, for example, a Chinese layout change doesn't cascade failure to the German page.
-
-* To be merged, translation PR's require a Website WG +1 and a +1 from another native speaker in your language. Make sure whoever you have review the PR adds a +1 in the comments of it so we know it is translated properly.
+> Since Apr, 1 2020 translation process moved to [Crowdin](https://crowdin.com/project/nodejs-website)
```
Comment (Trott) [inline TRANSLATION.md:7]: Probably can just leave this line out?

File: TRANSLATION.md [hunk 2]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+## Get started
+
+1. Open [nodejs-website](https://crowdin.com/project/nodejs-website) Crowdin project
+2. Find your locale and start translation. Find more details in [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/)
+
+All translated and approved content will be pushed to this repo automatically, you don't need to create any PRs with translation. Just keep localization process on Crowdin.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
All translated and approved content will be pushed to this repo automatically. You don't need to create any PRs with translation. Just keep localization process on Crowdin.

```

File: TRANSLATION.md [hunk 3]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/master/locale/en). If you found any problem with original source, please create PR with changes directly to this folder. Crowdin automatically pull all updates in 24 hours.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/master/locale/en). If you find any problem with original source, please create a PR with changes directly to `/locale/en`. Crowdin automatically pull all updates within 24 hours.
```

File: TRANSLATION.md [hunk 4]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+### Can't find my locale on Crowdin
+
+Please create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Crowdin managers team would be happy to add new language.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
Please create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Crowdin managers team would be happy to add new languages. 
```

File: TRANSLATION.md [hunk 5]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/master/locale/en). If you found any problem with original source, please create PR with changes directly to this folder. Crowdin automatically pull all updates in 24 hours.
+
+### Can't find my locale on Crowdin
+
+Please create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Crowdin managers team would be happy to add new language. 
+
+## Localization groups
+
+Localization group it's **NOT MUST HAVE** to start translation, and you can simply contribute on Crowdin without it. Just think about groups like a basement to communicate with other translators of your locale.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
An existing localization group is not required to start translation. You can contribute on Crowdin without it. Think about groups like a basement to communicate with other translators of your locale.
```

File: TRANSLATION.md [hunk 6]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+Contact your appropriate localization group, and discuss with them the best possible way to contribute. A list of the localization groups can be found here (please keep in mind that some groups have not yet taken action upon the project name changes, hence the outdated repository names):
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
Contact your appropriate localization group, and discuss with them the best possible way to contribute. A list of the localization groups can be found below.
```

Not sure what you mean about outdated repository names?

File: TRANSLATION.md [hunk 7]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+* [`nodejs-sv`](https://github.com/nodejs/nodejs-sv) Swedish Community
+* [`nodejs-ta`](https://github.com/nodejs/nodejs-ta) Tamil Community
+* [`nodejs-tr`](https://github.com/nodejs/nodejs-tr) Turkish Community
+* [`nodejs-zh-TW`](https://github.com/nodejs/nodejs-zh-TW) Taiwanese Community
+* [`nodejs-uk`](https://github.com/nodejs/nodejs-uk) Ukrainian Community
+* [`nodejs-vi`](https://github.com/nodejs/nodejs-vi) Vietnamese Community
+
+### Group for my locale is not exist
+
+If you can't find group for your locale, Node.js team would be happy to create it:
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
If you can't find group for your locale:
```

File: TRANSLATION.md [hunk 8]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+* [`nodejs-ro`](https://github.com/nodejs/nodejs-ro) Romanian Community
+* [`nodejs-ru`](https://github.com/nodejs/nodejs-ru) Russian Community
+* [`nodejs-sv`](https://github.com/nodejs/nodejs-sv) Swedish Community
+* [`nodejs-ta`](https://github.com/nodejs/nodejs-ta) Tamil Community
+* [`nodejs-tr`](https://github.com/nodejs/nodejs-tr) Turkish Community
+* [`nodejs-zh-TW`](https://github.com/nodejs/nodejs-zh-TW) Taiwanese Community
+* [`nodejs-uk`](https://github.com/nodejs/nodejs-uk) Ukrainian Community
+* [`nodejs-vi`](https://github.com/nodejs/nodejs-vi) Vietnamese Community
+
+### Group for my locale is not exist
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
### Group for my locale does not exist
```

File: TRANSLATION.md [hunk 9]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+If you can't find group for your locale, Node.js team would be happy to create it:
+
+1. Translate 1000 strings or more on Crowdin for your locale
+2. Find at least one more translator for your locale
+3. Create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo
+
+### Group for my locale is archived
+
+If you find group for your locale is archived, Node.js team would be happy to unarchive it:
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
If you find the group for your locale is archived:
```

File: TRANSLATION.md [hunk 10]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+* [`nodejs-uk`](https://github.com/nodejs/nodejs-uk) Ukrainian Community
+* [`nodejs-vi`](https://github.com/nodejs/nodejs-vi) Vietnamese Community
+
+### Group for my locale is not exist
+
+If you can't find group for your locale, Node.js team would be happy to create it:
+
+1. Translate 1000 strings or more on Crowdin for your locale
+2. Find at least one more translator for your locale
+3. Create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
3. Create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo requesting the creation of a group for your locale
```

File: TRANSLATION.md [hunk 11]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+
+### Group for my locale is archived
+
+If you find group for your locale is archived, Node.js team would be happy to unarchive it:
+
+1. Try to contact with members of this group by creating a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Add mentions for current members of this group
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
1. Try to contact members of the group by creating a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Include a mention of the group so members get notified of the issue.
```

File: TRANSLATION.md [hunk 12]
```diff
@@ -4,68 +4,73 @@ Node.js is a global platform and so this site has many translations. The transla
...
+2. If there is no response from members in 7 days, Node.js team would be happy to unarchive repo, if you already done 1000 strings or more on Crowdin for your locale.
```
Comment (Trott) [inline TRANSLATION.md]:
```suggestion
2. If there is no response from members in 7 days and if you have already done 1000 strings or more on Crowdin for your locale, open an issue in https://github.com/nodejs/admin requesting the repository be unarchived.
```

**Review by Trott — APPROVED** [review]
I left a bunch of suggestions, but this looks good to me with or without my suggestions.

File: TRANSLATION.md (hunk 1 above)
Comment (alexandrtovmach) [reply TRANSLATION.md:7]: I've added it as an explanation for current translators, who translate through GitHub flow, to not confuse

**Comment by alexandrtovmach** [comment]
@Trott Thanks for review and fixes to my grammar mistakes, committed your suggestions

File: TRANSLATION.md [hunk 13]
```diff
@@ -4,68 +4,74 @@ Node.js is a global platform and so this site has many translations. The transla
...
+> Since Apr, 1 2020 translation process moved to [Crowdin](https://crowdin.com/project/nodejs-website)
+
+## Get started
+
+1. Open [nodejs-website](https://crowdin.com/project/nodejs-website) Crowdin project
+2. Find your locale and start translation. Find more details in [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/)
+
+All translated and approved content will be pushed to this repo automatically. You don't need to create any PRs with translation. Just keep localization process on Crowdin.
+
+
```
Comment (alexandrtovmach) [inline TRANSLATION.md]:
```suggestion
```

**Pull request merged**